plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/CDKOpenshiftUtility.java

```diff
@@ -77,6 +78,8 @@ public IConnection createOpenshiftConnection(IServer server, ServiceManagerEnvir
 		((Connection)con).setRememberPassword(true);
 		
 
+		String defaultImageRegistry = "https://hub.openshift.rhel-cdk.10.1.2.2.xip.io ";
```

Comment:
nitpicking: move to constant so that it can be more easily identified as hard-coded?